java-frontend/src/main/java/org/sonar/java/classpath/Version.java

```diff
@@ -0,0 +1,125 @@
+/*
```

Comment:
We have an issue with this class, as it's not part of the public API but the `DependencyVersionAware` depends on it, it will be missing in the classpath for the analysis at runtime if a custom plugin tries to implement the interface.
I think we should also move the `Version` object to the public API.